### DISCUSSION TITLE
Make Sure That Free Security Features Are Activated (MSTG-CODE-9) - Add static encryption check

### DISCUSSION THREAD

**Discussion by rsenet** [OWASP/mastg#2352] [Ideas]
In https://mas.owasp.org/MASTG/iOS/0x06i-Testing-Code-Quality-and-Build-Settings/#static-analysis_7, one check is currently missing regarding the static analysis of the application: checking if the application is encrypted or not

- Encrypted 

```
otool -l TikTok | grep -A4 LC_ENCRYPTION_INFO
       cmd LC_ENCRYPTION_INFO_64
   cmdsize 24
  cryptoff 54919168
 cryptsize 4096
   cryptid 0
```

The output shows that the application is not encrypted (cryptid = 0).

**Comment by cpholguera** [comment]
Thanks @rsenet I'm adding an explicit mention of cryptid in the section for obtaining and decrypting IPAs. It'll be merged soon.
